src/views/Workflow.vue

```diff
@@ -227,10 +304,17 @@ export default {
         if (this.deltaSubscriptions.length === 0) {
           this.$workflowService.stopDeltasSubscription()
           this.tree.clear()
+        } else if (vm.deltaTableSubscriptions.includes(subscriptionId)) {
```

Comment:
Hmmm, this is never executed I'm afraid.

Here on GH UI, click on the button icon above to see a few more lines of the code above. You should see that this code is wrapped within a `if (vm.deltaSubscriptions.includes(subscriptionId)) {`. But if this `subscriptionId` exists in `deltaTableSubscriptions`…